### PR TITLE
Add ci script for publisher.

### PR DESCRIPTION
**Pull request by rmccar**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+  depth: false
 
 services:
   - docker
 
 language: node_js
 
 node_js:
-  - "8"
+  - "10"
```
Comment (jonnyshaw89) [inline .travis.yml]: `eq-author` wont build on 10. i hit that issue yesterday

**Comment by jonnyshaw89** [comment]
🎉

**Comment by jonnyshaw89** [comment]
@rmccar I think you committed node_modules

**Comment by rmccar** [comment]
Oh yeah! Cheers ill sort out the git ignore

**Comment by jonnyshaw89** [comment]
@rmccar all the node modules are still committed

**Pull request merged**